etl/steps/data/grapher/worldbank_wdi/2025-01-24/wdi.meta.yml

```diff
@@ -0,0 +1,524 @@
+definitions:
+  # NOTE: we'd like to move this to snapshot, but it's impractical because there are
+  # tons of other downstream datasets that still use sources
+  wdi_origin: &wdi_origin
+    producer: World Bank
+    title: World Bank World Development Indicators
+    description: |-
+      The World Development Indicators (WDI) is the World Bank's primary collection of development indicators, compiled from officially-recognized international sources. It presents the most current and accurate global development data available and includes national, regional, and global estimates.
+
+      [Text from [World Bank data catalog](https://datacatalog.worldbank.org/dataset/world-development-indicators)]
+    citation_full: World Development Indicators, The World Bank (2023).
+    url_main: https://datacatalog.worldbank.org/search/dataset/0037712/World-Development-Indicators
+    url_download: http://databank.worldbank.org/data/download/WDI_csv.zip
+    date_accessed: '2023-05-29'
+    date_published: '2023-05-11'
+    license:
+      name: CC BY 4.0
+      url: https://www.worldbank.org/en/about/legal/terms-of-use-for-datasets
```

Comment:
That one, or https://datacatalog.worldbank.org/search/dataset/0037712/World-Development-Indicators